packages/dendron-cli/src/commands/pod.ts

```diff
@@ -169,16 +177,25 @@ export function enrichPodArgs(opts: {
       });
     }
 
-    // if query is specified, then override config to pass in query
-    if (args.query) {
-      if (podType === "publish") {
+    if (podType === "publish") {
+      switch (podId) {
+        case MarkdownPublishPod.id:
+        case JSONPublishPod.id:
+        case HTMLPublishPod.id:
+          cleanConfig["dest"] = "stdout";
+      }
+      // if vault is specified, then override config to pass in
+      if (args.vault) {
+        cleanConfig["vaultName"] = args.vault;
+      }
+      if (args.query) {
         cleanConfig["fname"] = args.query;
-      } else {
-        // eslint-disable-next-line no-console
-        console.log(
-          `WARN: --query parameter not implemented for podType ${podType}`
-        );
       }
+    } else {
+      // eslint-disable-next-line no-console
+      console.log(
+        `WARN: --query and --vault parameter not implemented for podType ${podType}`
```

Comment:
the base command has a `print` method which you can use to avoid the eslint disable every time.

Reply:
are there pod types that don't actually require / implement these options?

Reply:
yeah, --query is not required by any pod except `publish` and export pods have `vaults` config option which is of type object.

Reply:
updated console.logs with print [f5d96d6](https://github.com/dendronhq/dendron/pull/2352/commits/f5d96d6fba56f55ea0b78fe468db14226961893d)